features/object_mapper/README.md

```diff
@@ -0,0 +1,15 @@
+# Object Mapper
+
+Version 2.1 of the driver introduces a simple object mapper, which avoids most of the boilerplate when converting your domain classes to and from query results.
+
+The object mapper handles basic CRUD operations in Cassandra tables containing UDTs, collections and all native CQL types.
+
+This documentation is organized in subsections defining all functionalities of the object mapper.
+
+The examples in this section are based on the following table structure:
+
+```java
+CREATE TABLE user (user_id uuid PRIMARY KEY, name text);
+CREATE TYPE address (street text, zip_code int);
+CREATE TABLE userAd (user_id uuid PRIMARY KEY, name text, ad address);
```

Comment:
It's a bit confusing to have two tables relating to the same concepts (users).
For the sake of clarity I would use something different for the second table, for example:
```
CREATE TABLE company (company_id uuid PRIMARY KEY, name text, address address);
```